doc/links.txt

```diff
@@ -30,9 +30,9 @@
 .. _`Cellular Monitor`: https://docs.nordicsemi.com/bundle/nrf-connect-cellularmonitor/page/managing_credentials.html
 .. _`nRF Connect Serial Terminal`: https://docs.nordicsemi.com/bundle/nrf-connect-serial-terminal/page/index.html
 
-.. _Radio Driver section: https://infocenter.nordicsemi.com/topic/15.4_radio_driver_v1.10.0/rd_release_notes.html
+.. _`Radio Driver section`: https://infocenter.nordicsemi.com/topic/15.4_radio_driver_v1.10.0/rd_release_notes.html
```

Comment:
Look like this link doesn't work anymore. 
This section is referenced in a Changelog for 802.15.4 Radio Driver for NCS release 1.6.0.